### PR TITLE
i18n(id): Update reference/frontmatter.md translation

### PR DESCRIPTION
**Pull request by iyyafi**
#### Description
- Update ID translation for reference/frontmatter.md file

### CODE REVIEW

**Review by HiDeoo — CHANGES_REQUESTED** [review]
Thanks for the translation :raised_hands:

As a non-native speaker, I reviewed this PR using the [Docs Translation Review Checklist](https://hackmd.io/1KYnrpUQR0SAGTzHIUoUDg?view). Left a suggestion that should fix the link issue but otherwise looks good to me :thumbsup:

File: docs/src/content/docs/id/reference/frontmatter.md [hunk 1]
```diff
@@ -344,10 +386,79 @@ Atribut HTML untuk ditambahkan ke tautan halaman di _sidebar_ saat ditampilkan d
...
+};
+```
+
+Pelajari selengkapnya tentang koleksi skema konten di [“Mendefinisikan koleksi skema”](https://docs.astro.build/en/guides/content-collections/#defining-a-collection-schema) di dokumentasi Astro.
+
+`docsSchema()` memiliki opsi berikut:
+
+### `extend`
+
+**tipe:** Skema Zod atau fungsi yang mengembalikan skema Zod
```
[the remark this answers is not in the document]
Comment (iyyafi) [reply docs/src/content/docs/id/reference/frontmatter.md]: Ah i see, i will commit this suggestion. thank you.

File: docs/src/content/docs/id/reference/frontmatter.md [hunk 2]
```diff
@@ -241,11 +262,27 @@ Mengatur apakah halaman ini harus disertakan dalam indeks pencarian [Pagefind](h
...
 pagefind: false
 ---
 ```
 
+### `draft`
+
+**tipe:** `boolean`  
+**bawaan:** `false`
+
+Tetapkan apakah halaman ini harus dianggap sebagai draf dan tidak disertakan dalam [versi produksi](https://docs.astro.build/en/reference/cli-reference/#astro-build) dan [grup tautan yang dibuat secara otomatis](/id/guides/sidebar/#autogenerated-groups). Ubah ke `true` untuk menandai halaman sebagai draf dan membuatnya hanya terlihat selama masa pengembangan.
```
[the remark this answers is not in the document]
Comment (iyyafi) [reply docs/src/content/docs/id/reference/frontmatter.md]: Thank you, i also commit this suggestion

**Review by HiDeoo — APPROVED** [review]
Thanks for updating the PR, looks perfect to me now :rocket:

**Pull request merged**